var/spack/repos/builtin/packages/ghostscript/package.py

```diff
@@ -29,14 +29,23 @@ class Ghostscript(Package):
     """An interpreter for the PostScript language and for PDF."""
 
     homepage = "http://ghostscript.com/"
-    url = "http://downloads.ghostscript.com/public/old-gs-releases/ghostscript-9.18.tar.gz"
```

Comment:
I tried using the new URL, but `spack versions` does not find any remote versions for ghostscript. I think this is because github does not store tarballs in a common directory, and the directories are not listable anyway. Do you know if there is a URL format that will allow `spack versions` to show the available releases on github?